src/status_im/popover/core.cljs

```diff
@@ -9,6 +9,7 @@
    :dismiss-keyboard nil})
 
 (fx/defn hide-popover
-  {:events [:hide-popover]}
+  {:events [:hide-popover
+            :reset-popover]}
```

Comment:
why do we need two different names?

Reply:
oh.. this is only for developer experience
you see, when implementing local popover anywhere using the global popover component, I noticed that when I make change, and figwheel reloads, then I click the local popover, nothing happens. Unless I go to another locality and trigger popover there, come back and trigger the popover in the locality I'm working on.

I found out that it is because when figwheel reloads the old popover in :popover/popover is still there and when I trigger local popover no state change and no effect

So I made it less annoying

Reply:
yeah but better to make temporary changes for development and do not commit them

Reply:
understood and removed